backend/tests/test_project.py

```diff
@@ -71,3 +71,46 @@ def test_project_routes(self):
             data=json.dumps(update_obj),
         )
         self.assertEqual(response.status_code, 404)
+
+    # test get_projects endpoint
+    def test_get_project(self):
```

Comment:
@peterghrong Is it possible for you to change `test_project_routes` (a few lines above, Github won't let me comment there for some reason) to `test_update_project`? As now the naming would be a bit misleading with the new changes, which is a great idea to separate them so that we know what's failing and what's not instead of bundling all of them together in a massive fail. TYSM, should have foresaw that.

Reply:
fixed